app/logic/transcript.py

```diff
@@ -20,7 +20,7 @@ def getOtherEventsTranscript(username):
                     .join(Program, JOIN.LEFT_OUTER).switch(Event)
                     .join(Term)
                     .where(EventParticipant.user==username,
-                    Event.program == None)
+                    Event.program == 9)
```

Comment:
This where clause is short enough to all be on the same line, also we should not be using a magic number.